tests/integration/components/o-s-s/attribute-text-test.ts

```diff
@@ -0,0 +1,53 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import sinon from 'sinon';
+import click from '@ember/test-helpers/dom/click';
+
+module('Integration | Component | o-s-s/attribute-text', function (hooks) {
+  setupRenderingTest(hooks);
+
+  module('Default behavior', function () {
+    test('it renders', async function (assert) {
+      await render(hbs`<OSS::Attribute::Text @label="Hello" @value="World" />`);
+      assert.dom('.oss-attribute--text').exists();
+    });
+
+    test('it renders without a value', async function (assert) {
+      await render(hbs`<OSS::Attribute::Text @label="Hello" />`);
+      assert.dom('.oss-attribute--text').exists();
+    });
```

Comment:
The `assert` is the same, why writing 2 different tests? Test the real value rendered

Reply:
This one is fine IMO, maybe change the label to clarify:
`It renders when the @value is undefined` (can be done with the empty state)